tests/src/Integration/RetrofitTitleResolverTest.php

```diff
@@ -45,13 +46,17 @@ public function testSetStoredTitle(): void
             RetrofitTitleResolver::class,
             $titleResolver
         );
+        $requestStack = $this->container->get('request_stack');
+        assert($requestStack instanceof RequestStack);
+        $request = $requestStack->getCurrentRequest();
+        assert($request instanceof Request);
```

Comment:
Huh, I'm surprised this works without pushing a request to the stack.

Reply:
I found that `pathInfo` is not set when a request is created, so it never matched the request from the request stack.